sirepo/job_supervisor.py

```diff
@@ -303,6 +308,20 @@ def __db_init_history(self, prev_db):
             PKDict(((k, v) for k, v in prev_db.items() if k in _HISTORY_FIELDS)),
         ]
 
+    @classmethod
+    def __db_load(cls, compute_jid):
+        def _fixup_fields(fields, db):
```

Comment:
I don't think this needs to be a function. It will be less code to inline the `if`.